### ISSUE TITLE
Linting is in there twice

### ISSUE THREAD

**Issue by dret** [isaqb-org/curriculum-api#38]
I am not sure that this is a problem but right now we have linting in there two times. It is both managed in the API lifecycle management chapter (chapter 6) and the API governance chapter (chapter 8). It does make sense in both places, so maybe it's best to leave in in both places?

**Comment by sippsack** [comment]
That is fine for me. Let's postpone this discussion after we have some feedback of the first trainings.

**Comment by dret** [comment]
Makes sense and as I mentioned, I do think it is relevant in both cases.
